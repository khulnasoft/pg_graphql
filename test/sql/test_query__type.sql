select graphql.resolve(
    query:='query Abc { __type(name: "Int") { name kind description } }'
);
