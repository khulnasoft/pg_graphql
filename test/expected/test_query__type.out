select graphql.resolve(
    query:='query Abc { __type(name: "Int") { name kind description } }'
);
                                                 resolve                                                  
----------------------------------------------------------------------------------------------------------
 {"data": {"__type": {"kind": "SCALAR", "name": "Int", "description": "A scalar integer up to 32 bits"}}}
(1 row)

